php_phongo.c

```diff
@@ -193,34 +193,56 @@ void phongo_throw_exception(php_phongo_error_domain_t domain, const char* format
 	va_end(args);
 }
 
+static int phongo_exception_append_error_labels(zval* labels, const bson_iter_t* iter)
```

Comment:
If I understand correctly, you split this logic into a separate function since it is called for both `errorLabels` and `writeConcernError.errorLabels`.

Reply:
Correct 👍